dev_tools/version_test.py

```diff
@@ -25,7 +25,8 @@ def test_versions_are_the_same():
 def _get_version(package: str):
     version_file = f'{package}/_version.py'
     resulting_locals: Dict[str, Any] = {}
-    exec(open(version_file).read(), globals(), resulting_locals)
+    with open(version_file, encoding="utf8") as file:
+        exec(file.read(), globals(), resulting_locals)
```

Comment:
`_get_version` is a dead code as of [#4157](https://github.com/quantumlib/Cirq/pull/4157/files#diff-28d298ed5a4d675d94d0d120db53453b356a5dac954f0d7dd747c4bfeea7b0bf).

Let us delete it.  (nice to have codecov to catch this)

Reply:
Thanks for catching that. I've removed it in the latest commits on this PR branch.